packages/realm-server/handlers/handle-stripe-webhook.ts

```diff
@@ -14,10 +15,23 @@ export default function handleStripeWebhookRequest({
     let response = await stripeWebhookHandler(
       dbAdapter,
       request,
-      async (stripeUserId: string) => {
-        let user = await getUserByStripeId(dbAdapter, stripeUserId);
-        if (user) {
-          await sendEvent(user.matrixUserId, 'billing-notification');
+      async ({
```

Comment:
Why does this need to be a callback? Can it just be a normal function in `packages/billing/stripe-webhook-handlers/index.ts`?

Reply:
I see it's needed because it need the `this.matrixClient` reference which lives in the `RealmServer`